netebpfext/user/fwp_um.cpp

```diff
@@ -166,6 +166,7 @@ _fwp_engine::test_cgroup_inet4_recv_accept(_In_ fwp_classify_parameters_t* param
         const_cast<FWP_BYTE_BLOB*>(&parameters->app_id);
     incoming_value[FWPS_FIELD_ALE_AUTH_RECV_ACCEPT_V4_ALE_USER_ID].value.byteBlob =
         const_cast<FWP_BYTE_BLOB*>(&parameters->user_id);
+        incoming_value[FWPS_FIELD_ALE_CONNECT_REDIRECT_V4_FLAGS].value.uint32=  FWP_CONDITION_FLAG_IS_REAUTHORIZE;
```

Comment:
```suggestion
        incoming_value[FWPS_FIELD_ALE_CONNECT_REDIRECT_V4_FLAGS].value.uint32 =  FWP_CONDITION_FLAG_IS_REAUTHORIZE;
```
nit: Make spacing consistent throughout
(I'm somewhat surprised that clang-format didn't do this automatically)

Reply:
Unresolving since only this occurrence was touched, not "throughout".  All the other lines added have the same spacing issue.

Reply:
updated the code.